cmd/releasego/testdata/update-azure-linux/cgmanifest.json

```diff
@@ -1,35 +1,45 @@
 {
   "Registrations": [
-      {
-        "component": {
-          "type": "other",
-          "other": {
-            "name": "golang",
-            "version": "1.22.3",
-            "downloadUrl": "https://github.com/microsoft/go/releases/download/v1.22.3-1/go1.22.3-20240507.3.src.tar.gz"
-          }
+    {
+      "component": {
+        "type": "other",
+        "other": {
+          "name": "golang",
+          "version": "1.22.2",
+          "downloadUrl": "https://github.com/microsoft/go/releases/download/v1.22.3-1/go1.22.3-20240507.3.src.tar.gz"
         }
-      },
-      {
-        "component": {
-          "type": "other",
-          "other": {
-            "name": "abseil-cpp",
-            "version": "20240116.0",
-            "downloadUrl": "https://github.com/abseil/abseil-cpp/archive/refs/tags/20240116.0.tar.gz"
-          }
+      }
+    },
+    {
+      "component": {
+        "type": "other",
+        "other": {
+          "name": "golang",
+          "version": "1.23.0",
+          "downloadUrl": "https://github.com/microsoft/go/releases/download/v1.22.3-1/go1.22.3-20240507.3.src.tar.gz"
```

Comment:
URL does not match is that on purpose for this test?

Reply:
It doesn't matter, so I'll update it.